master/static/migrations/20240814123944_add-view-resource-quotas-permissions.tx.up.sql

```diff
@@ -0,0 +1,13 @@
+/* Add RBAC permissions for creating/updating/deleting namespace-workspace bindings and 
+resource quotas. */
+INSERT into permissions(id, name, global_only) VALUES 
+    (11003, 'view resource quotas', false);
+
+INSERT INTO permission_assignments(permission_id, role_id) VALUES
+    (11003, 1),
+    (11003, 2),
+    (11003, 4),
```

Comment:
I noticed that we don't have `(11003, 3)`, where `role_id == 3` is the Workspace Creator role. Do we specifically not want workspace creators to have this permission? If so, how come

Reply:
Ooh okok that makes sense, thanks!